### PR TITLE
Client copy update

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/forms/qae_2014_forms/innovation/innovation_step3.rb [hunk 1]
```diff
@@ -79,8 +79,13 @@ def innovation_step3
 
         header :company_financials, "Company Financials" do
           context %(
-            <p>These figures should be for your entire organisation. If you haven't reached your latest year-end, please use estimates to complete this section.</p>
-                    )
+            <p>
+              A parent company making a group entry should include the trading figures of all UK members of the group.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step3.rb]: Line is too long. [118/100]

File: app/forms/qae_2014_forms/innovation/innovation_step3.rb [hunk 2]
```diff
@@ -79,8 +79,13 @@ def innovation_step3
...
+            </p>
+            <p>
+              If you haven't reached your latest year-end, please use estimates to complete this section.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step3.rb]: Line is too long. [105/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step2.rb [hunk 3]
```diff
@@ -55,7 +55,7 @@ def trade_step2
           ref "B 3"
           required
           context %{
-            <p>Include, for example, comparisons between domestic and international strategies, treatment of different markets (linking to top performing markets), market research, market development, routes to market, after sales and technical advice, staff language training, export practices, overseas distributors, inward/outward trade missions, trade fairs and visits to existing/potential markets. Make sure you explain how your actual performance compares to your planned performance.</p>
+            <p>Include, for example, comparisons between domestic and international strategies, treatment of different markets (linking to top performing markets), market research, market development, routes to market, after sales and technical advice, activities to sustain/grow markets, staff language training, export practices, overseas distributors, inward/outward trade missions, trade fairs and visits to existing/potential markets. Make sure you explain how your actual performance compares to your planned performance.</p>
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step2.rb]: Line is too long. [531/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step3.rb [hunk 4]
```diff
@@ -80,8 +80,13 @@ def trade_step3
 
         header :company_financials, "Company Financials" do
           context %(
-            <p>These figures should be for your entire organisation. If you haven't reached your latest year-end, please use estimates to complete this section.</p>
-                    )
+            <p>
+              A parent company making a group entry should include the trading figures of all UK members of the group.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step3.rb]: Line is too long. [118/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step3.rb [hunk 5]
```diff
@@ -80,8 +80,13 @@ def trade_step3
...
+            </p>
+            <p>
+              If you haven't reached your latest year-end, please use estimates to complete this section.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step3.rb]: Line is too long. [105/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb [hunk 6]
```diff
@@ -77,8 +77,13 @@ def development_step3
 
         header :company_financials, "Company Financials" do
           context %(
-            <p>These figures should be for your entire organisation. If you haven't reached your latest year-end, please use estimates to complete this section.</p>
-                    )
+            <p>
+              A parent company making a group entry should include the trading figures of all UK members of the group.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb]: Line is too long. [118/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb [hunk 7]
```diff
@@ -77,8 +77,13 @@ def development_step3
...
+            </p>
+            <p>
+              If you haven't reached your latest year-end, please use estimates to complete this section.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step3.rb]: Line is too long. [105/100]

File: app/models/eligibility/development.rb [hunk 8]
```diff
@@ -16,7 +16,7 @@ class Eligibility::Development < Eligibility
 
   property :development_contributed_to_commercial_success,
             boolean: true,
-            label: "Has your sustainable development had a positive impact on your commercial success over the last two years?",
+            label: "Has your sustainable development had a positive impact on your commercial success in the last two years?",
```
Comment (matthewford) [inline app/models/eligibility/development.rb]: Line is too long. [126/100]

File: app/models/eligibility/promotion.rb [hunk 9]
```diff
@@ -31,6 +31,7 @@ class Eligibility::Promotion < Eligibility
   property :contributed_to_enterprise_promotion_within_uk,
             boolean: true,
             label: "Has their contribution to enterprise promotion been within the UK (including the Channel Islands and the Isle of Man)?",
+            hint: "Foreign nationals are eligible for the award as long as their contribution was within the UK.",
```
Comment (matthewford) [inline app/models/eligibility/promotion.rb]: Line is too long. [114/100]

File: app/models/eligibility/trade.rb [hunk 10]
```diff
@@ -8,33 +8,35 @@ class Eligibility::Trade < Eligibility
 
   property :sales_above_100_000_pounds,
             values: %w[yes no skip],
-            label: "Have you had £100,000 or more overseas sales in each of the last three years?",
+            label: "Have you made a minimum of £100,000 in overseas sales in each year of your entry (i.e. in the last 3 or 6 years)?",
```
Comment (matthewford) [inline app/models/eligibility/trade.rb]: Line is too long. [135/100]

File: app/models/eligibility/trade.rb [hunk 11]
```diff
@@ -8,33 +8,35 @@ class Eligibility::Trade < Eligibility
...
-            accept: :true,
-            acts_like_boolean: true,
-            if: proc { !direct_overseas_100_000_pounds? }
+  # REMOVE
+  # property :direct_overseas_100_000_pounds,
+  #           boolean: true,
+  #           values: %w[yes no],
+  #           label: "Was at least £100,000 of this direct overseas sales?",
+  #           accept: :not_nil,
+  #           hint: "Direct overseas sales are as a result of an organisation making a commitment to market overseas on its own behalf - and not through an intermediary."
```
Comment (matthewford) [inline app/models/eligibility/trade.rb]: Line is too long. [170/100]

File: app/models/eligibility/trade.rb [hunk 12]
```diff
@@ -8,33 +8,35 @@ class Eligibility::Trade < Eligibility
...
+  #           values: %w[yes no],
+  #           label: "Do your organisation fulfill any of the exceptions above?",
+  #           accept: :true,
+  #           acts_like_boolean: true,
+  #           if: proc { !direct_overseas_100_000_pounds? }
+  # REMOVE
 
   property :any_dips_over_the_last_three_years,
-            label: "Have you had any dips in your overseas sales over the last three years?",
+            label: "Have you had any dips in your overseas sales over the period of your entry (i.e. in the last 3 or 6 years)?",
```
Comment (matthewford) [inline app/models/eligibility/trade.rb]: Line is too long. [129/100]<br>Align the parameters of a method call if they span more than one line.

File: app/models/eligibility/trade.rb [hunk 13]
```diff
@@ -8,33 +8,35 @@ class Eligibility::Trade < Eligibility
...
             accept: :false,
             boolean: true,
             hint: "For this question a 'dip' is any annual drop in overseas sales.",
             if: proc { !skipped? }
 
   property :growth_over_the_last_three_years,
-            label: "Have you had substantial growth in overseas earnings over the last three years?",
+            label: "Have you had substantial growth in overseas earnings in the last three years?",
```
Comment (matthewford) [inline app/models/eligibility/trade.rb]: Align the parameters of a method call if they span more than one line.

File: spec/models/eligibility/trade_spec.rb [hunk 14]
```diff
@@ -55,7 +55,11 @@
     let(:eligibility) { Eligibility::Trade.new(account: account) }
 
     it 'returns all questions for new eligibility' do
-      expect(eligibility.questions).to eq([:sales_above_100_000_pounds, :direct_overseas_100_000_pounds, :organisation_fulfill_above_exceptions, :any_dips_over_the_last_three_years, :growth_over_the_last_three_years])
+      expect(eligibility.questions).to eq([:sales_above_100_000_pounds,
+                                          # :direct_overseas_100_000_pounds,
+                                          # :organisation_fulfill_above_exceptions,
+                                          :any_dips_over_the_last_three_years,
```
Comment (matthewford) [inline spec/models/eligibility/trade_spec.rb]: Align the elements of an array literal if they span more than one line.

File: spec/models/eligibility/trade_spec.rb [hunk 15]
```diff
@@ -55,7 +55,11 @@
...
+                                          :growth_over_the_last_three_years])
```
Comment (matthewford) [inline spec/models/eligibility/trade_spec.rb]: Align the elements of an array literal if they span more than one line.

**Comment by RuslanHamidullin** [comment]
All good for me :+1:

**Pull request merged**